core/templates/dev/head/pages/error/Error.js

```diff
@@ -22,5 +22,15 @@ oppia.controller('Error', [
       $scope, UrlInterpolationService) {
     $scope.oopsMintImgUrl = UrlInterpolationService.getStaticImageUrl(
       '/general/oops_mint.png');
+
+    $scope.ErrorCode = GLOBALS.errorCode;
```

Comment:
Follow naming conventions in the file -- we don't use UpperCamelCase for simple variables. Should be $scope.errorCode.